Tests/UITextViewSpecs.swift

```diff
@@ -0,0 +1,182 @@
+import Quick
+import Nimble
+import ViewConfigurator
+
+class UITextViewSpecs: QuickSpec {
+    
+    override func spec() {
+        it("can set text") {
```

Comment:
I would envelop all tests in a describe function `describe("UIViewConfigurator") {` this will improve the test names, to read like a sentence.